### ISSUE TITLE
implement mega-comparison

### ISSUE THREAD

**Issue by hdiamondlowe** [zkbt/mosasaurus#25]
For now [Cube.py](https://github.com/zkbt/mosasaurus/blob/d28c5a10b3fc7eeba6ccaea9fd45081d427e1b4b/mosasaurus/Cube.py#L417) can only make a cube with one target star and one comparison star. We should be able to have multiple comparison stars!

**Comment by zkbt** [comment]
Yes, for sure! To do that, we’d simply need to construct a mega-star to represent the comparison. Do you have the time/inclination to try to tackle this, or would you like me to take a look?

-------_--------_--------_--------_-------

Zach Berta-Thompson
Assistant Professor, Exoplaneteer
(pronouns = he/him/his)

Department of Astrophysical & Planetary Sciences
University of Colorado at Boulder
casa.colorado.edu/~bertathompson<http://casa.colorado.edu/~bertathompson>
+1(303)735-6821

I stand with undocumented students, staff, and their families.

-------_--------_--------_--------_-------

On Jun 24, 2019, at 3:42 PM, hdiamondlowe <notifications@github.com<mailto:notifications@github.com>> wrote:


For now Cube.py<https://github.com/zkbt/mosasaurus/blob/d28c5a10b3fc7eeba6ccaea9fd45081d427e1b4b/mosasaurus/Cube.py#L417> can only make a cube with one target star and one comparison star. We should be able to have multiple comparison stars!

**Comment by hdiamondlowe** [comment]
I'm actually debating if the mega-star should be crated in `mosasaurus`, or downstream in [`decorrasaurus`](https://github.com/hdiamondlowe/decorrasaurus). My thinking is that if `mosasaurus` could just hold multiple, separate comparison stars, then it can be decided in `decorrasaurs` which to use. Otherwise I have to re-create the cube every time I want to exclude/include a star.

What are your thoughts on this? I'm not sure how else you are using `mosasaurus` so I don't want to mess with the outputs without your okay.

**Comment by zkbt** [comment]
Oh, yes, totally. I think it'd make way more sense to implement it in `decorrasaurus`. Most of the `Cube.py` stuff was mostly for making quick light curves and visualizations. Since you're inevitably going to be doing fancy fitting, just do it there. It'll make it easier to keep track of all the measurements to which you want to have access for all the stars.
